llm_bench/python/requirements.txt

```
@@ -1,10 +1,10 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 numpy
-openvino>=2024.1.0
+openvino~=2024.2.0
 auto-gptq>=0.5.1 # for gptq
 pillow
 torch
-transformers>=4.39.0
+transformers>=4.39.0,<4.41.0
```

fix for compatibility with transformers 4.41 provided in release branch
```suggestion
transformers>=4.39.0,<4.42.0
```

---

updated